projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -770,13 +770,13 @@ public static function enqueue_block_editor_assets() {
 		}
 
 		wp_localize_script(
-			'jetpack-blocks-editor',
+			'jetpack-publicize',
```

Comment:
Same, if this doesn't break everything else I think it's just by luck.

Reply:
Same as the other response.